docs/commands/push.md

```diff
@@ -1,93 +1,186 @@
-```
+#### API name
+
+To find your API name required for the command:
+
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. Check the list of APIs displayed on this page.
+1. Inspect the title of each list item to the left of the **New version** and **Edit API** action buttons. This title is an API name.
+
+When using the `push` command, you would provide API name after the [Organization ID](#organization-id) separated with the forward slash (`/`). For example, `@test_docs/petstore-api`.
+
+:::info
+The name of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+:::
 
+#### API version
 
-The name and version of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+To find your API version required for the command:
 
-If the `branchName` option is omitted, the command will use the default branch.
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. Check the list of APIs displayed on this page.
+1. Inspect the sub-title of each list item to the bottom of the [API name](#api-name). This sub-title is an API version.
+
+When using the `push` command, you would provide API version after the [API name](#api-name) separated with the "at" symbol (`@`). For example, `@test_docs/petstore-api@v1`.
+
+:::info
+The version of your API definition should contain only supported characters (`a-z`, `A-Z`, `0-9`, `-`, `.`). Using a restricted character will result in an error, and your API definition will not be created.
+:::
+
+### Run-id
 
 The `--run-id` option can be used by Redocly Workflows to associate multiple pushes with a single CI job. It is auto-populated for the following CI systems so you don't have to pass it separately:
 
 - Travis CI
 - CircleCI
 - GitHub Actions
 
-
-### Set up CI from Redocly Workflows
+#### Set up CI from Redocly Workflows
 
 The Redocly Workflows interface can help you get started with the `push` command.
 
 1. In **API Registry**, select **Add API**.
+1. In the **Definition name** step, provide a name for your new API definition.
+1. In the **Choose source** step, select **Upload from CI/CD**. This will generate syntax for the `push` command that you can copy and use to upload a new API definition file. Alternatively, use the `openapi push -u` command directly from the command-line interface.
+
+### Upsert
+
+To create a new API and a new API definition version with the `push` command, use the `--upsert` or `-u` option:
+
+```bash
+openapi push -u test-api-v1.yaml @redocly/test-api@v1 main
+```
+
+:::warning
+Before executing the `push` command with the `--upsert` option, ensure that the corresponding organization (that you are supposed to push API definitions to) exists in your Redocly API Registry
+:::
+
+## References
```

Comment:
Maybe this would work better if we called it "Learn more" or "See the push command in action".

"References" would be OK if we had mutiple links to pages with more (technical) details about the command, or some specific guides...but we don't.